homework/vector-of-shared-ptrs/vectorFunctions.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include "vectorFunctions.hpp"
+#include <iostream>
+
+std::vector<std::shared_ptr<int>> vector_ptrs;
```

Comment:
move to function
Reserve memory when you know how many elements will be inside vector.